save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CreationView.kt

```diff
@@ -224,23 +225,20 @@ class CreationView : RComponent<PropsWithChildren, ProjectSaveViewState>() {
         val gitUser = fieldsMap[InputTypes.GIT_USER]
         if (gitUser.isNullOrBlank() || gitUser.trim().matches(".*\\s.*")) {
             setState { isValidGitUser = false }
-            valid = false
```

Comment:
this validation is not needed as it is optional, will make a better validation in a different issue